components/emails/slack-integration-notification.tsx

```
@@ -0,0 +1,84 @@
+            </Text>
+            <Text className="max-w-sm flex-wrap break-words font-medium text-purple-600 no-underline">
+              {settingsUrl.replace(/^https?:\/\//, "")}
+            </Text>
+            <Hr />
+            <Section className="mt-8 text-gray-400">
+              <Text className="text-xs">
+                © {new Date().getFullYear()}{" "}
+                <a
+                  href="https://www.papermark.com"
+                  className="text-gray-400 no-underline hover:text-gray-400"
+                  target="_blank"
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)